packages/api/cms-api/package.json

```
@@ -131,7 +131,7 @@
         "@nestjs/graphql": "^13.0.0",
         "@nestjs/jwt": "^11.0.0",
         "@nestjs/platform-express": "^11.0.0",
-        "@sentry/node": "^8.0.0",
+        "@sentry/node": "^8.0.0 || ^9.0.0",
```

Do we want to support both versions btw? Probably sufficient to just keep v9

```suggestion
        "@sentry/node": "^9.0.0",
```

---

I would only support v9.

---

I'm fine with both, so let's only support v9 😁